Makefile

```diff
@@ -6,11 +6,11 @@ PYTHONPATH := `pwd`
 # Install uv so we can do everything with it
 .PHONY: setup
 setup:
-	pip install uv
+	curl -LsSf https://astral.sh/uv/install.sh | sh
 
 .PHONY: install
 install:
-	uv pip install -e .
+	uv sync
```

Comment:
This isn't really necessary, but it's nice to have the venv created if we want to select its Python as the interpreter in an IDE.